ui/tests/integration/components/shamir/dr-token-flow-test.js

```diff
@@ -3,12 +3,15 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
+/* eslint-disable qunit/require-expect */
```

Comment:
Is this still necessary? I noticed that the test below does include `expect(2)` 🤔

Reply:
The rule is sort of badly named -- ref https://github.com/platinumazure/eslint-plugin-qunit/blob/main/docs/rules/require-expect.md

But, it's not actually enforced until ember 5 so I'll remove it since it was linting from leftover package modules